pkg/provision/providers/qemu/node.go

```diff
@@ -84,9 +84,6 @@ func (p *provisioner) createNode(state *vm.State, clusterReq provision.ClusterRe
 	// Talos config
 	cmdline.Append("talos.platform", constants.PlatformMetal)
 
-	// SELinux
-	cmdline.Append("selinux", "1")
-
 	// add overrides
 	if nodeReq.ExtraKernelArgs != nil {
 		if err = cmdline.AppendAll(nodeReq.ExtraKernelArgs.Strings()); err != nil {
```

Comment:
can we instead use negative merging here and pass `-selinux` in case of apparmor test?

Reply:
I would rather leave `selinux=1` enabled by default for now

Reply:
so check if we pass `apparmor=1`, then add `-selinux`?

Reply:
actually I got it, better way

Reply:
I mean in the `e2e-qemu.sh` directly if apparmor was requrest

Reply:
yeh, did the same